src/azul/transformer.py

```diff
@@ -167,7 +176,12 @@ def translate_fields(cls,
                         new_dict[key + '_'] = val
             return new_dict
         elif isinstance(doc, list):
-            return [cls.translate_fields(val, field_types, forward=forward) for val in doc]
+            # Translate [] to [null_string] but only at the field level (to avoid case of contents['organoids'] == [])
+            if len(doc) == 0 and not isinstance(field_types, dict):
```

Comment:
```suggestion
            if doc or isinstance(field_types, dict):
```

and swap the if-else branches.